source/_integrations/plugwise.markdown

```diff
@@ -2,12 +2,12 @@
 title: Plugwise
 description: Plugwise Smile platform integration.
 ha_category:
+  - Binary Sensor
   - Climate
   - Sensor
   - Switch
-  - Water Heater
 ha_iot_class: Local Polling
-ha_release: 0.98
```

Comment:
We should keep this at `0.98` as this is 'included since'

Reply:
Ok, will change back.